### DISCUSSION TITLE
10. LeetCode 题解

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#183] [Comments]
# 10. 题目解析

超详细的「算法与数据结构」基础讲解教程，从零基础开始学习算法知识，800+ 道「LeetCode 题目」详细解析，200 道「大厂面试热门题目」

[https://algo.itcharge.cn/solutions/](https://algo.itcharge.cn/solutions/)

**Comment by hunanikun** [comment]
萌新打卡
